lldb/include/lldb/Symbol/Type.h

```diff
@@ -300,6 +303,12 @@ class TypeQuery {
       m_options &= ~e_find_one;
   }
 
+  /// Returns true if the type query is supposed to treat the name to be
+  /// searched as a mangled name.
+  bool GetSearchByMangledName() const {
+    return (m_options & e_search_by_mangled_name) != 0;
+  }
```

Comment:
Is there a setter that only lives on the Apple fork then? Don't think it'd be too outrageous to add both the getter/setter here as we do for all the other options (even if we only have coverage for it on the Apple fork i guess)